### PR TITLE
Added logo the login component

### PR DESCRIPTION
**Pull request by plagzee**
### What does this PR change?

- It doesn't changes any functionality of the app. Trying to do my first contributions. Saw a issue popping "enhancement" which stated to add a large paralus logo to the main login page. So I decided to do it by using simple jsx and tailwind. The size of the iamge that loads is in 4k, checked in other throttle modes. Works fine.

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- https://github.com/paralus/dashboard/issues/110

### Checklist

I confirm, that I have...

- [x] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [x] Formatted the code using `npm run format` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [x] Updated `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 - Hide copy button on non https webpages from [meain](https://github.com/meain)
 - Removed building images to registry on pull requests from [niravparikh05](https://github.com/niravparikh05)
+- Added logo to the main login page
```
Comment (meain) [inline CHANGELOG.md:11]: Could you also add the `from ...` bit in there.

File: src/containers/Auth/Login/index.js [hunk 2]
```diff
@@ -250,6 +250,7 @@ class Login extends Component {
     return (
       <>
         <>
+          <img src="/logolarge.png" className="login-logo img-fluid mb-4" />
```
Comment (meain) [inline src/containers/Auth/Login/index.js:253]: Would it be possible for you to add it to the forgot password page as well?

**Review by meain — COMMENTED** [review]
Hey @plagzee , thanks for looking into it.

For anyone else looking into it, below is how it looks.



@niravparikh05 , should we add some text as well in here? Also, probably change the button text from `Next` to `Login`?

**Comment by plagzee** [comment]
Hello @meain, I'll take a look of what you've mentioned tomorrow. I'll add the changes you mentioned.

**Comment by plagzee** [comment]
may I know the url and query that makes the forgot password page working? And also the file where the forgot password components are exported as I cannot find it in Auth.js. So, I can take a look on the UI change you mentioned on the forgot password page

**Comment by meain** [comment]
https://github.com/RafayLabs/rcloud-console-ui/blob/main/src/containers/Auth/KratosSettings.js is where you will find the password reset page which is loaded when you navigate to `/ksettings`. You can view this in the application when you have to reset your password (you will have to go through this when you create a new user).

**Comment by niravparikh05** [comment]
lgtm, yeah let's change the text from Next to Login.

**Comment by plagzee** [comment]
### check this out

image

**Comment by techmaharaj** [comment]
> ### check this out
> 
> image

Let's fix the type-case also in this since we are working on this. The 'P' in both passwords placeholder text needs to be in caps i.e. "Password"

**Comment by plagzee** [comment]
> > ### check this out
> > image
> 
> Let's fix the type-case also in this since we are working on this. The 'P' in both passwords placeholder text needs to be in caps i.e. "Password"

image

**Comment by meain** [comment]
Hey @plagzee , looks like when this gets built and served via docker the paths are a bit different. The current images that we have in `public` are only used the manifest, rest of the images are coming from `src/assets`. Could you change use of images to something like below. You'll also have to move the images to the proper dir. Maybe rename the `alt` to "Paralus logo" as well I guess.

```diff
diff --git a/src/containers/Auth/KratosSettings.js b/src/containers/Auth/KratosSettings.js
index c6fe1f2..af8a801 100644
--- a/src/containers/Auth/KratosSettings.js
+++ b/src/containers/Auth/KratosSettings.js
@@ -8,6 +8,7 @@ import { useQuery } from "utils/helpers";
 import PageLayout from "./Login/components/PageLayout";
 import { createTheme, MuiThemeProvider } from "@material-ui/core/styles";
 import tealTheme from "../themes/tealTheme";
+import paralusLogo from "assets/images/logolarge.png"
 
 const KratosSettings = (props) => {
   const { query } = useQuery();
@@ -102,7 +103,7 @@ const KratosSettings = (props) => {
             autoComplete="off"
             onSubmit={handleChangePassword}
           >
-            <img src="/logolarge.png" alt="logo" className="img-fluid" />
+            <img src={paralusLogo} alt="logo" className="img-fluid" />
             <div style={{ marginTop: "25px", marginBottom: "25px" }}>
               <div className="row mt-4">
                 <div className="col-md-12">
diff --git a/src/containers/Auth/Login/index.js b/src/containers/Auth/Login/index.js
index 6c2ab88..8fc25f8 100644
--- a/src/containers/Auth/Login/index.js
+++ b/src/containers/Auth/Login/index.js
@@ -28,6 +28,7 @@ import { Redirect, withRouter } from "react-router-dom";
 import PageLayout from "./components/PageLayout";
 import ChangePasswordForm from "./components/ChangePasswordForm";
 import { newKratosSdk } from "actions/Auth";
+import paralusLogo from "assets/images/logolarge.png"
 
 const SESSION_TIMEOUT_MILLISECONDS = 12 * 60 * 60 * 1000 + 1 * 60 * 1000; // 12hrs + 1min
 class Login extends Component {
@@ -251,7 +252,7 @@ class Login extends Component {
       <>
         <>
           <img
-            src="/logolarge.png"
+            src={paralusLogo}
             alt="logo"
             className="login-logo img-fluid mb-4"
           />
```

**Comment by plagzee** [comment]
@meain done

**Review by meain — APPROVED** [review]
LGTM, thank you! 🎉

**Pull request merged**